OrbitClientData/include/OrbitClientData/ModuleData.h

```diff
@@ -42,7 +42,9 @@ class ModuleData final {
   // gone, remove the parameter here
   void AddSymbols(const orbit_grpc_protos::ModuleSymbols& module_symbols,
                   uint64_t module_base_address);
-  void ClearSymbols();
+  // Same as last comment: As soon as FunctionInfo does not contain module_base_address anymore,
+  // completely remove the following method
```

Comment:
Maybe add `TODO(antonrohr)` in the beginning of the line as well?

Reply:
Probably, makes no sense as it's obvious. Feel free to ignore.

Reply:
I added a bug and used the bug number